test/test_cuda.py

```diff
@@ -76,6 +76,13 @@ def tearDown(self):
         del self.autocast_lists
         super().tearDown()
 
+    def test_pinned_memory_with_cudaregister(self):
+        os.environ["PYTORCH_CUDA_ALLOC_CONF"] = "pinned_use_cuda_host_register:True,pinned_num_register_threads:8"
```

Comment:
This should be `torch.cuda.memory._set_allocator_settings` otherwise it won't have an effect. Make sure to set it back after the test.